### DISCUSSION TITLE
Wormhole

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#115] [Games]
# Wormhole

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/wormhole/?giscus=3b991498fd6b413b36192e3eM7Gm%2B9QB9sriGonGRVKIEanHO%2F4hHQRHCkv%2B1Q7%2FTBIHJARoh5%2Fut7zkqzUEl%2FP3kQ%2BKugJvs0%2Bo0%2Fin%2BmHSKXIz%2ByJRhRu0hb6sno9M3RrL%2BVXsUdy%2FKnonNSU%3D

**Comment by thedavesims** [comment]
Created for GBJAM 9 using C. [Source Code](https://github.com/thedavesims/Wormhole).
